main.js

```diff
@@ -49,11 +49,17 @@ $(function(){
 		} else {
 			formHTML += '<input type="'+$input.attr('type')+'" class="input-text input-full" name="'+id+'" id="'+id+'">';
 		}
+		var buttonLabel = $input.attr('data-button-label');
```

Comment:
@noahdklein buttonLabel shouldn't be an attribute of the input, it should be on the $emailContainer object.
To simplify this I'd condense into one line added after line 5 (to keep the data attrs pulled from the container together):

`var buttonLabel = ($emailContainer.attr('data-button-label') != 'undefined') ? $emailContainer.attr('data-button-label') : 'Submit'; `